sdk/batch/Azure.ResourceManager.Batch/api/Azure.ResourceManager.Batch.netstandard2.0.cs

```diff
@@ -529,46 +606,94 @@ protected MockableBatchSubscriptionResource() { }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Batch.Models.AccessRule>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Batch.Models.AccessRule System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Batch.Models.AccessRule>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Batch.Models.AccessRule>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Batch.Models.AccessRule>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct AccessRuleDirection : System.IEquatable<Azure.ResourceManager.Batch.Models.AccessRuleDirection>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public AccessRuleDirection(string value) { throw null; }
+        public static Azure.ResourceManager.Batch.Models.AccessRuleDirection Inbound { get { throw null; } }
+        public static Azure.ResourceManager.Batch.Models.AccessRuleDirection Outbound { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Batch.Models.AccessRuleDirection other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Batch.Models.AccessRuleDirection left, Azure.ResourceManager.Batch.Models.AccessRuleDirection right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Batch.Models.AccessRuleDirection (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Batch.Models.AccessRuleDirection left, Azure.ResourceManager.Batch.Models.AccessRuleDirection right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class AccessRuleProperties : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Batch.Models.AccessRuleProperties>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Batch.Models.AccessRuleProperties>
```

Comment:
The name is too generic.

Reply:
updated in new PR https://github.com/Azure/azure-sdk-for-net/pull/45847